keycloak/themes/uid2-theme/login/login-update-password.ftl

```diff
@@ -70,6 +66,43 @@
                     </#if>
                 </div>
             </div>
+
+
+        <script type="text/javascript">
+            let blacklistedPasswords = [];
+
+            function loadBlacklist() {
+                // txt file of common passwords recommended to blacklist by NIST
+                fetch('https://raw.githubusercontent.com/danielmiessler/SecLists/master/Passwords/Common-Credentials/10-million-password-list-top-1000000.txt')
```

Comment:
pulling from this file, but we can also store the txt file in our code if that makes sense too.  i went with this option because i didnt want to take up space in the code for a huge file, but it would be an issue if this URL changes

Reply:
I don't love relying on a 3rd party to be up in order for our blacklist to work.  You can just save the file locally and load it from there, too.